library/Rules/Callback.php

```diff
@@ -13,26 +13,45 @@
 
 namespace Respect\Validation\Rules;
 
-use Respect\Validation\Exceptions\ComponentException;
+use function array_shift;
+use function array_unshift;
+use function call_user_func_array;
+use function func_get_args;
 
-class Callback extends AbstractRule
+/**
+ * This is a wildcard validator, it uses a function name, method or closure.
+ *
+ * @author Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author William Espindola <oi@williamespindola.com.br>
+ */
+final class Callback extends AbstractRule
```

Comment:
I want put more information relevante on this properties and constructor param but I can't figure out some way to not be obvious like `@var callable $callback Callable statement that will be called`